mithril-common/src/crypto_helper/tests_setup.rs

```diff
@@ -246,7 +246,11 @@ pub fn setup_certificate_chain(
                     .unwrap()
                 }
                 _ => {
-                    fake_certificate.metadata.signers = fixture.signers_with_stake();
+                    fake_certificate.metadata.signers = fixture
+                        .signers_with_stake()
+                        .into_iter()
+                        .map(|s| -> StakeDistributionParty { s.into() })
+                        .collect();
```

Comment:
It would be nice to migrate this code into a `fn stake_distribution_parties()` method on the `MithrilFixture`.